[Docs] task distribution (#5246)
recognai/rubrix: argilla/docs/how_to_guides/dataset.md

!!! tip
    If you want further guidance on good practices for guidelines during the project development, check our [blog post](https://argilla.io/blog/annotation-guidelines-practices/).

### Distribution

> When you are working as a team, you may want to distribute the annotation task to ensure efficiency and quality. You ca use the `TaskDistribution` settings to configure the number of minimum submitted responses expected for each record. Argilla will use this setting to automatically handle records in the pending queues of your team members.

Reviewer: ```suggestion
When working as a team, you may want to distribute the annotation task to ensure efficiency and quality. You can use the `TaskDistribution` settings to configure the number of minimum submitted responses expected for each record. Argilla will use this setting to automatically handle records in your team members' pending queues.
```
